modules/installation-configuration-parameters.adoc

```diff
@@ -734,19 +734,23 @@ Not all CCO modes are supported for all cloud providers. For more information ab
 If your AWS account has service control policies (SCP) enabled, you must configure the `credentialsMode` parameter to `Mint`, `Passthrough` or `Manual`.
 ====
 |`Mint`, `Passthrough`, `Manual` or an empty string (`""`).
-ifndef::openshift-origin,ibm-power-vs[]
-|`fips`
-|Enable or disable FIPS mode. The default is `false` (disabled). If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
+// ifndef::openshift-origin,ibm-power-vs[]
+// |`fips`
+// |Enable or disable FIPS mode. The default is `false` (disabled). If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
+////
```

Comment:
Why are we hiding this one and not using the admonition?

Reply:
In cases where FIPS can be removed from inline text, tables, etc we are doing so. The admonition was only used in cases  where code samples (mainly our install config sample files) could not be easily revised to removed the reference.